src/systemctl/systemctl.c

```diff
@@ -488,6 +492,7 @@ static int output_units_list(const UnitInfo *unit_infos, unsigned c) {
                                 printf("%.*s\n", desc_len, "DESCRIPTION");
                         else
                                 printf("%s\n", "DESCRIPTION");
+                        printf("%s", ansi_normal());
```

Comment:
It's probably a good idea to move the `\n` into this `printf()` call as well, as place it *after* the ANSI sequence, no? At least in "systemd-mount --list" the underline only goes until the rightmost *used* column and not any further.